index.html

```diff
@@ -2,38 +2,64 @@
 <html>
 <head>
     <meta charset="utf-8">
-    <title>What should I do?</title>
-    <link rel="stylesheet" href="style.css">
+    <title>Abbas Mashhouri Component</title>
+    <link rel="stylesheet" href="css/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
 </head>
-    <body>
-        <script type="text/x-handlebars" data-template-name="todos">
-            <section id="todoapp">
-                <header id="header">
-                    <h1>The TODO list</h1>
-                </header>
+<body>
+    <h1 class="mainTitle"> Task Component</h1>
+    <script type="text/x-handlebars" data-template-name="Abs">
+        <section id="TasksSection">
+            <header id="header">
+                <input type="checkbox" id="chk-all" {{action "checkAll" on="change" }}>
+                {{input type="text" id="new-Tasks" placeholder="Add Task here (type then enter)"
+                value=newTitle action="createTask"}}
+            </header>
+            <section id="main">
+                <ul id="Tasks-list">
+                    {{#each Task in model itemController="Tasks"}}
+                    <li {{bind-attr class="Task.isDone:completed Task.editMode:editing" }}>
+                        {{#if Task.editMode}}
+                        {{input type="text" value=Task.title class="edit" data-id=Task.id focus-out="changeTask" insert-newline="changeTask"}}
+                        {{else}}
+                        {{input type="checkbox" checked=Task.isDone class="chk" }}
+                        <label {{action "editTasks" on="doubleClick" }}>{{Task.title}}</label><button {{action "removeTask" }} class="remove"></button>
+                        {{/if}}
+                    </li>
+                    {{/each}}
+                </ul>
 
-                <section id="main">
-                </section>
-
-                <footer id="footer">
-                </footer>
             </section>
-
-            <footer id="info">
+            <footer id="footer">
+                <span id="Tasks-count">
+                    <strong>{{remainCount}}</strong> task(s) left
+                </span>
+                <ul id="filters">
```

Comment:
No, it's not nice code, because if we use several of this component in one page , it has conflict Id,   however I wanted to create this component at a short time, and I thought it's for test not for use. its better to create dynamic id or change these id to class if we need it. 


     On Monday, September 28, 2015 5:15 PM, Ruan Naude <notifications@github.com> wrote:
   

 In index.html:>              </section>
> -
> -            <footer id="info">
> +            <footer id="footer">
> +                <span id="Tasks-count">
> +                    <strong>{{remainCount}}</strong> task(s) left
> +                </span>
> +                <ul id="filters">
Do you think it is a good idea to use the id tag everywhere?—
Reply to this email directly or view it on GitHub.